src/exception.cr

```diff
@@ -147,7 +156,7 @@ end
 #
 # This can be used either to stub out method bodies, or when the method is not
 # implemented on the current platform.
-class NotImplementedError < Exception
+class NotImplementedError < LogicError
```

Comment:
If platform-specific behavior is not implemented then I don't think that counts as the user's fault.

Reply:
My thinking was that if your program is consistently getting a `NotImplementedError`, then maybe you should do something to gracefully handle it. But I don't have strong feeling either way, I'll leave this one the way it was.